conf.yaml

```diff
@@ -197,11 +197,13 @@ contents:
                 repo:   x-pack-elasticsearch
                 prefix: elasticsearch-extra/x-pack-elasticsearch
                 path:   docs/en
+                private: true
```

Comment:
This change will require that I manually kick off a jenkins build with the `rebuild` option after merging this.